bulbs/utils/test/__init__.py

```diff
@@ -54,7 +53,7 @@ def default(self, value):
         iso = _iso_datetime(value)
         if iso:
             return iso
-        if not PY3 and isinstance(value, str):
+        if PY2 and isinstance(value, str):
```

Comment:
PY3+ best practice, better to check PY2, so we don't break on future PY4 (which will be more similar to 3 than 2)